ui/src/conferences/containers/DetailPageContainer.jsx

```diff
@@ -23,25 +23,11 @@ import { newSearch } from '../../actions/search';
 import { CONFERENCE_CONTRIBUTIONS_NS } from '../../reducers/search';
 import DeletedAlert from '../../common/components/DeletedAlert';
 import { makeCompliantMetaDescription } from '../../common/utils';
+import withRouteDataFetcher from '../../common/withRouteDataFetcher';
 
-function DetailPage({ loading, match, dispatch, record }) {
-  const recordId = match.params.id;
-
-  useEffect(
-    () => {
-      dispatch(fetchConference(recordId));
-      dispatch(newSearch(CONFERENCE_CONTRIBUTIONS_NS));
-      window.scrollTo(0, 0);
-    },
-    [dispatch, recordId]
-  );
-
+function DetailPage({ record }) {
```

Comment:
what about the ``scrollTo`` stuff?

Reply:
this is in my notes, only thing missing in the PR right now. I think it's safe to `scrollTo(0)`, whenever pathname changes, what do you think?